pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md

```diff
@@ -14,7 +14,7 @@ After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCl
 
 ## Example deployment
 
-<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
+<p class="message--note"><strong>NOTE: </strong>If you need to manage these custom resources across all clusters in a project, it is recommended you use [Project Deployments](../../../../../project-deployments) which enables you to leverage GitOps to deploy the resources. Otherwise, you will need to create the custom resources manually in each cluster.</p>
```

Comment:
hmm, for some reasons the link is not working when I try running local dev. it displays as `[Project Deployments](../../../../../project-deployments)`. maybe within the html tags it just ignore markdown syntax? similar for the other 2 apps

Reply:
good catch! let me fix that